packages/cli/src/types.ts

```diff
@@ -0,0 +1,126 @@
+// Reference to and optional definition of a type alias:
+// export type ${name} = ${definition}
+import { keyof } from 'io-ts';
+
+export class TypeAlias {
+  constructor(
```

Comment:
What do you think about adding an `inline: boolean` option here?
This will allow to inline core primitive type instead of reexporting them as aliases.
I think it makes sense to not alias `number`, `string` and `boolean` to reduce potential confusion for new users.

Reply:
I think I might be missing your meaning of inline. But if you omit the definition then no declaration is emitted so we don't provide some alias for 'boolean' etc. The idea is if you don't provide the optional declaration it works just like the old enum version did - then it just acts like a typed constant. I was thinking that a type name is the noop version of a TypeAlias.

Reply:
Ah, right, disregard this. For some reason I thought all type aliases are reexported.